src/main/java/com/bookstore/model/Book.java

```diff
@@ -18,18 +25,34 @@
 @Table(name = "books")
 public class Book {
 
-    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
+
     @Column(nullable = false)
     private String title;
+
     @Column(nullable = false)
     private String author;
+
     @Column(nullable = false, unique = true)
     private String isbn;
+
     @Column(nullable = false)
     private BigDecimal price;
+
     private String description;
+
     private String coverImage;
+
+    @ManyToMany(cascade = CascadeType.REMOVE, fetch = FetchType.EAGER)
```

Comment:
please avoid `EAGER` fetch type. please read [here](https://stackoverflow.com/a/24154042) why you shouldn't use `EAGER`